README.md

```diff
@@ -63,18 +64,17 @@ helm install nuodb-incubator/<chart>
 
 This GitHub repository contains the source for the packaged and versioned charts released in the [`gs://nuodb-charts` Google Storage bucket](https://console.cloud.google.com/storage/browser/nuodb-charts/) (the Chart Repository).
 
-The Charts in the `stable/` directory in the master branch of this repository match the latest packaged Chart in the Chart Repository, though there may be previous versions of a Chart available in that Chart Repository.
+The Charts in this repository are organized into two folders:
 
-The purpose of this repository is to provide a place for maintaining and contributing official Charts, with CI processes in place for managing the releasing of Charts into the Chart Repository.
 
-The Charts in this repository are organized into two folders:
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [stable](stable/README.md)
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [incubator](incubator)
 
-* stable
-* incubator
 
-Stable Charts meet the criteria in the [technical requirements](CONTRIBUTING.md#technical-requirements).
+The charts in the `stable` directory in the master branch of this repository match the latest packaged Chart in the Chart Repository, though there may be previous versions of a Chart available in that Chart Repository. The purpose of this repository is to provide a place for maintaining and contributing official Charts, with CI processes in place for managing the releasing of Charts into the Chart Repository.
 
-Incubator Charts are those that do not meet these criteria. Having the incubator folder allows charts to be shared and improved on until they are ready to be moved into the stable folder. The charts in the `incubator/` directory can be found in the [`gs://nuodb-charts-incubator` Google Storage Bucket](https://console.cloud.google.com/storage/browser/nuodb-charts-incubator).
+The charts in the `Incubator` directory are those that do not meet these criteria. Having the incubator folder allows charts to be shared and improved on until they are ready to be moved into the stable folder. The charts in the `incubator/` directory can be found in the [`gs://nuodb-charts-incubator` Google Storage Bucket](https://console.cloud.google.com/storage/browser/nuodb-charts-incubator).
 
 In order to get a Chart from incubator to stable, Chart maintainers should open a pull request that moves the chart folder.
```

Comment:
we should remove this sentence. It makes sense for 3rd party products, but we are the maintainers of said charts.

Reply:
Done.